test/DebuggerCommon/async_step_over.js.dbg.baseline

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "callStack": [
+      {
+        "line": 9,
+        "column": 4,
+        "sourceText": "return await af1(10)",
+        "function": "af2"
+      },
+      {
+        "line": 12,
+        "column": 0,
+        "sourceText": "var p = af2()",
+        "function": "Global code"
+      }
+    ]
+  },
+  {
+    "callStack": [
+      {
+        "line": 28,
+        "column": 0,
+        "sourceText": "p.then(result => {\r\n        if (result === 100) {\r\n            print(\"PASS\");\r\n        }\r\n    },\r\n    error => {\r\n        print(\"Failed : \" + error);\r\n    } \r\n)",

```

Comment:
curious why it is not truncated. Did we change the default, or ch has different default?